src/medium-zoom.js

```diff
@@ -379,6 +379,10 @@ const mediumZoom = (selector, options = {}) => {
         // image best fitting the current viewport size, respecting the `srcset`.
         active.zoomedHd.removeAttribute('sizes')
 
+        // In Firefox, the `loading` attribute needs to be set to `eager` for the
+        // load event to be fired.
+        active.zoomedHd.setAttribute('loading', 'eager')
```

Comment:
Would it make sense to let the browser chose the default value by using `''`?

Reply:
It seems to work as well. Changed in a new commit.